src/averageAges.js

```diff
@@ -37,7 +46,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => withChildren
+    ? people.find(mother => person.name === mother.mother)
```

Comment:
it's better to use here `.some()` cause we don't need object that returns the find a method